tests/unit/render/test_vega.py

```diff
@@ -284,51 +281,19 @@ def test_metric_missing(tmp_dir, scm, dvc, caplog):
     assert first(plot_content["layer"])["encoding"]["y"]["field"] == "y"
 
 
-def test_custom_template(tmp_dir, scm, dvc, custom_template):
-    metric = [{"a": 1, "b": 2}, {"a": 2, "b": 3}]
```

Comment:
test_templates::test_load_template is responsible for testing custom template load now